### PR TITLE
tools: adding reminders for API shephards

### PR DESCRIPTION
**Pull request by alyssawilk**
Doing daily digests for non-maintainer-shephards where the PR is 1) not a draft and 2) needs API LGTM ('pending' status)

cc @markdroth @adisuissa

### CODE REVIEW

File: .github/actions/pr_notifier/pr_notifier.py [hunk 1]
```diff
@@ -98,37 +129,38 @@ def track_prs():
         # SLO, nudge in bold if not.
         message = pr_message(delta, pr_info.html_url, pr_info.title, delta_days, delta_hours)
 
+        if (needs_api_review(labels, repo, pr_info)):
+            add_reminders(pr_info.assignees, api_review_and_prs, message, API_REVIEWERS)
```
Comment (asraa) [inline .github/actions/pr_notifier/pr_notifier.py:133]: is it ever the case that an api PRs are unassigned? I'm pretty sure repokitteh handles that by random assignment anyway though.. so i think that's ok

File: .github/actions/pr_notifier/pr_notifier.py [hunk 2]
```diff
@@ -98,37 +129,38 @@ def track_prs():
...
         # If there was no maintainer, track it as unassigned.
         if not has_maintainer_assignee:
-            # don't bother assigning maintainer WIPs.
-            if pr_info.draft and pr_info.user.login in maintainers_and_prs.keys():
-                continue
             maintainers_and_prs['unassigned'] = maintainers_and_prs['unassigned'] + message
 
     # Return the dict of {maintainers : PR notifications}, and stalled PRs
-    return maintainers_and_prs, stalled_prs
+    return maintainers_and_prs, api_review_and_prs, stalled_prs
```
Comment (asraa) [inline .github/actions/pr_notifier/pr_notifier.py:148]: might want to update the comment about this with the return val, but non-blocking

**Review by asraa — APPROVED** [review]
LGTM otherwise! I was battling a bit with commenting "why not let `track_pr()` return a single map of people to remind instead of one for all PRs/one for API PRs", but I think this is fine for separabilitiy

File: .github/actions/pr_notifier/pr_notifier.py (hunk 2 above)
Comment (alyssawilk) [reply .github/actions/pr_notifier/pr_notifier.py:148]: Good call - will do in a follow-up, as it's late to do today and I'd like to get the blocking bits checked in before tomorrow's run :-)

File: .github/actions/pr_notifier/pr_notifier.py (hunk 1 above)
Comment (alyssawilk) [reply .github/actions/pr_notifier/pr_notifier.py:133]: Yeah, it should always be assigned but we could arguably find it not having an API reviewer if someone removed themselves for vacation.  We could have a list for unassigned_api - I'll consider it as a feature if it ever happens or @envoyproxy/api-shepherds tell me it's worth it :-)

**Pull request merged**